guides/gatherers.md

```diff
@@ -66,3 +67,30 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+
+### package_version
+
+This gatherer allows checking the installation of a certain package in the system
```

Comment:
I would rephrase to a more specific description.
`This gatherer returns the version as a string of the specified package` or similar.